WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/StatsListViewModel.kt

```diff
@@ -144,7 +162,33 @@ class InsightsListViewModel
         analyticsTracker,
         dateSelectorFactory.build(INSIGHTS),
         popupMenuHandler
-)
+) {
+    override val uiModel: LiveData<UiModel> = super.uiModel.map { model ->
+        if (model is Success && model.data.none { item -> item.type == LOADING }) {
+            return@map Success(
+                    model.data + listOf(
+                            StatsBlock.Control(
+                                    LINK_BUTTON,
+                                    listOf(
+                                            LinkButton(
+                                                    R.string.edit,
+                                                    NavigationAction.create {
+                                                        mutableNavigationTarget.value = ViewInsightsManagement()
+                                                    }
+                                            )
+                                    )
+                            )
+                    )
+            )
+        } else {
+            return@map model
+        }
+    }
+
+    enum class InsightsTypes : StatsTypes {
```

Comment:
I'd remove the `StatsTypes` from the base `StatsBlock` implementation. I think the `Control` block doesn't need type. We just need to do a bit of casting in the `StatsBlockAdapter` and modify the `DiffUtils` so they don't check the statsTypes when it's comparing the `Control` block